pkg/i18n/english.go

```diff
@@ -1322,8 +1326,12 @@ func EnglishTranslationSet() TranslationSet {
 		CreateFixupCommitDescription:        `Create fixup commit`,
 		CreateFixupCommitTooltip:            "Create 'fixup!' commit for the selected commit. Later on, you can press `{{.squashAbove}}` on this same commit to apply all above fixup commits.",
 		SquashAboveCommits:                  "Apply fixup commits",
-		SquashAboveCommitsTooltip:           `Squash all 'fixup!' commits above selected commit (autosquash).`,
-		SureSquashAboveCommits:              `Are you sure you want to squash all fixup! commits above {{.commit}}?`,
+		SquashAboveCommitsTooltip:           `Squash all 'fixup!' commits, either above the selected commit, or all in current branch (autosquash).`,
+		SquashCommitsAboveSelectedTooltip:   `Squash all 'fixup!' commits above the selected commit (autosquash).`,
+		SquashCommitsInCurrentBranchTooltip: `Squash all 'fixup!' commits in the current branch (autosquash).`,
+		SquashCommitsInCurrentBranch:        "in current branch",
```

Comment:
This should use 'Sentence case' (likewise with below)

Reply:
OK, cc6dae41d4f1